Source/GoBot/FeedStrategy/HomeStrategy.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  HomeStrategy.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 19/12/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+/// This strategy acts like a Composite strategy that uses one strategy (selected by the user in the Settings screen)
+/// and routes all functions through that strategy so that we have the responsibility of choosing the right strategy
+/// to this class and let other parts o
+final class HomeStrategy: NSObject, FeedStrategy {
```

Comment:
This is a nice way to pull some functionality out of the Bot. And it is Sendable! 👍 👍 

I think we may have discussed this before, but we have two names going on related to these classes: `Strategy` and `Algorithm`. I think it could reduce confusion to use one or the other, and personally I like Algorithm more as it is a little more specific. How would feel about renaming FeedStrategy and related classes to FeedAlgorithm (not necessarily in this PR).

Reply:
@martindsq bump

Reply:
I created issue #1072 to address the naming and other things I think we can improve there